### ISSUE TITLE
1.1.0 breaks CommonJS imports

### ISSUE THREAD

**Issue by noomorph** [nebrelbug/npm-to-yarn#30]
**Describe the bug**

Cannot run this code: `require('npm-to-yarn')`.

```
Error [ERR_REQUIRE_ESM]: require() of ES Module /Users/yaroslavs/Projects/repro/node_modules/npm-to-yarn/dist/npm-to-yarn.js from /Users/yaroslavs/Projects/repro/[eval] not supported.
npm-to-yarn.js is treated as an ES module file as it is a .js file whose nearest parent package.json contains "type": "module" which declares all .js files in that package scope as ES modules.
Instead rename npm-to-yarn.js to end in .cjs, change the requiring code to use dynamic import() which is available in all CommonJS modules, or change "type": "module" to "type": "commonjs" in /Users/yaroslavs/Projects/repro/node_modules/npm-to-yarn/package.json to treat all .js files as CommonJS (using .mjs for all ES modules instead).
```

**To Reproduce**

```
mkdir repro
cd repro
npm init -y
npm install npm-to-yarn --save-dev
node -p "require('npm-to-yarn')"
```

**Expected behavior**

It should run without errors and return the `convert` function, like the version `1.0.1`.

```
 node -p "require('npm-to-yarn')"
[Function: convert]
```

**Package & Environment Details**

Tested on Node.js 14.x - 18.x

**Additional context**

I'm not well-versed in this entire CJS/MJS/ESM subject, but I think you should have released a major (breaking) version if this is unavoidable.

On the other side, if this breakage was rather accidental, maybe you could find a way to fix it for CommonJS users?

**Comment by bgub** [comment]
@noomorph no, this was accidental. I'm trying to update to the new NodeNext module resolution pattern. I'll try to fix this now.

**Comment by bgub** [comment]
@noomorph I just released version 1.2.1 with a `.cjs` build so it should work with CommonJS module resolution. Will you let me know if this resolves your problem?

**Comment by noomorph** [comment]
Yes 🙌 it works. Thanks!
